src/components/portfolio/DetailsEdit.tsx

```diff
@@ -49,23 +56,29 @@ const DetailsEdit: React.FC<DetailsEditProps> = ({
   id,
   name,
 }) => {
-  const classes = useStyles();
+  const classes = useStyles({ virtual });
   const { t } = useTranslation();
   const [openDuplicate, setOpenDuplicate] = React.useState<boolean>(false);
   const [openEdit, setOpenEdit] = React.useState(false);
 
   return (
     <div className={classes.subContainer}>
-      <Button
-        variant="contained"
-        className={classes.button}
-        onClick={() => setOpenEdit(true)}
-        disabled={!positions?.length || !virtual}
+      <Tooltip
+        title={
+          virtual ? '' : t('portfolio.details.cannotEditPortfolio').toString()
```

Comment:
I would add the disclaimer: 'Please duplicate to a virtual portfolio to simulate changes.'
I would also change styling to make the tooltip text a bit larger.